packages/shared-ux/chrome/navigation/src/ui/navigation.stories.tsx

```diff
@@ -551,3 +514,121 @@ export const CreativeUI = (args: NavigationServices) => {
     </NavigationWrapper>
   );
 };
+
+export const UpdatingState = (args: NavigationServices) => {
```

Comment:
I created this new Storybook for manually testing state updates to the active nodes. I wasn't really sure how to use Storybook actions or argTypes to do this: I updated the template component to show a button and put the click handler in the Storybook component to update state in the mocked services.

@clintandrewhall would you please take a look and let me know if you see a better way to do this?